doc/examples/example_x509.md

```diff
@@ -98,11 +98,10 @@ mococrw::X509Certificate cert;
 std::string certToPem = cert.toPEM();
 ```
 
-It's possible to add more parameters to the Certificate verification by providing the verify method
-with a VerificationCtx Object where we can set a CRL (certificate revocation list),
-a time check verification and others.
-In the example below, we can see an example where we create a context that enables
-a time check verification.
+It's possible to add more parameters to the certificate verification by providing the verify method
+with a VerificationCtx object where we can set, amongst others, a CRL (certificate revocation list) and
+a time-check verification.
```

Comment:
```suggestion
a time of verification.
```
From reading: Not quite sure what a time-check verification is. Probably means to specify the point in time for which the validity should be checked.

Reply:
How about a "time verification check"?

Reply:
sounds good to me

Reply:
Done.